### ISSUE TITLE
Use of JSON.stringify is problematic

### ISSUE THREAD

**Issue by oliversalzburg** [amekkawi/cwlogs-writable#7]
Related: https://github.com/mirkokiefer/bunyan-cloudwatch/issues/26

The library should use the same stringification approahces that bunyan itself uses. `JSON.stringify` is dangerous.

**Comment by oliversalzburg** [comment]
This seems useful:
```js
// Useful for custom `type == 'raw'` streams that may do JSON stringification
// of log records themselves. Usage:
//    var str = JSON.stringify(rec, bunyan.safeCycles());
module.exports.safeCycles = safeCycles;
```

**Comment by amekkawi** [comment]
Thanks @oliversalzburg. I'll make sure I'm handling cyclic references and also add some tests.

**Comment by amekkawi** [comment]
Fixed by [v0.2.0](https://github.com/amekkawi/cwlogs-writable/releases/tag/v0.2.0).
